### PR TITLE
Adding AMD and AMD Documentation

### LINKED ISSUE #1030: AMD - Automated Music Downloader
Any chance we can add AMD - Automated Music Downloader

Its a downloader that works with Lidarr

Release :: AMD - Automated Music Downloader (Lidarr Companion Docker)
Companion App
Docker Link: https://hub.docker.com/r/randomninjaatk/amd
Github Repo Link: https://github.com/RandomNinjaAtk/docker-amd
Unraid Community App Store: Available

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Closes #1030 

**Approach**
How does this change address the problem?

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [ ] Use github checklists. When solved, check the box and explain the answer.

**Learning**
Describe the research stage
Links to blog posts, patterns, libraries or addons used to solve this problem

**Requirements**
Check all boxes as they are completed

- [ ] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [ ] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -11,151 +11,152 @@ edit_uri: "https://github.com/GhostWriters/DockSTARTer/edit/master/docs/"
 copyright: "Copyright &copy; 2020 GhostWriters"
 
 markdown_extensions:
-  - meta
+    - meta
```
Comment (nemchik) [inline mkdocs.yml:14]: What's going on with all this? Why is it all indented extra now? (not saying it's wrong, but wondering why)

**Comment by nemchik** [comment]
Also please resolve merge conflicts. mkdocs had a new app added in your other PR that is missing here and can't auto-merge because of the indentation change.

File: mkdocs.yml (hunk 1 above)
Comment (angrycuban13) [reply mkdocs.yml:14]: Not sure how that got indented. I didn't even touch that part...

Comment (angrycuban13) [reply mkdocs.yml]: I think cbd5202 fixes it? But also ae7749a removed stuff 😭

**Comment by nemchik** [comment]
Finally got the labeler right.

**Pull request merged**